### PR TITLE
pmorales/nomics utils

### PR DESCRIPTION
**Pull request by pmm09c**
Adding the initial nomics util files and tests to the repository as well as some option directory structure

### CODE REVIEW

File: src/nomics_tests.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from dextrader.nomics.utils import get_candles, format_query_as_dataframe, get_recent_trades
+from APIKEYS import API_KEY
+import os
+
+
```
Comment (sububer) [inline src/nomics_tests.py:5]: Thanks for adding the tests, as this also helps illustrate usage.

Comment (pmm09c) [reply src/nomics_tests.py:5]: np!

File: src/dextrader/nomics/utils.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import urllib.request
+import json
+import os
+import pandas as pd
+
+
```
Comment (sububer) [inline src/dextrader/nomics/utils.py:6]: More a note for me when I get working on this.. but maybe adding either docstring and/or [typing hints](https://docs.python.org/3.7/library/typing.html#module-typing) to the function parameters for the hints to work in IDE/jupyter.

**Review by sububer — DISMISSED** [review]
Great start. Looks good so far.

File: src/dextrader/nomics/utils.py (hunk 2 above)
Comment (pmm09c) [reply src/dextrader/nomics/utils.py:6]: added typehints

Comment (sububer) [reply src/dextrader/nomics/utils.py:6]: Awesome!

**Review by sububer — APPROVED** [review]

**Pull request merged**